server/models/dtos/project_dto.py

```diff
@@ -129,9 +146,17 @@ def __hash__(self):
         if self.project_statuses:
             for project_status in self.project_statuses:
                 hashable_project_statuses = hashable_project_statuses + project_status
+        hashable_map_editors = ''
+        if self.map_editors:
+            for map_editor in self.map_editors:
+                hashable_map_editors = hashable_map_editors + map_editor
+        hashable_validate_editors = ''
```

Comment:
same: `hashable_mapping_editors` and `hashable_validation_editors`

Reply:
Fixed 23c4bb4